services/TS29222_CAPIF_Logging_API_Invocation_API/config.yaml

```diff
@@ -0,0 +1,12 @@
+mongo: {
+  'user': 'root',
+  'password': 'example',
+  'db': 'capif',
+  'col': 'invocationlogs',
```

Comment:
change the names col1, col2 etc to something more descriptive, invoker_col, log_col, etc